### PR TITLE
move to use dps orb

### PR DESCRIPTION
**Pull request by mattops**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,56 +1,11 @@
...
-      - checkout:
-          path: ~/git
-      - attach_workspace:
-          at: ~/app
-      - deploy:
-          name: Deploy to << parameters.env >>
-          command: |
-            APP_VERSION=$(egrep buildNumber build-info.json  | awk -F"\"" '{print $4}')
-            sed -i "s/appVersion: \".*\"/appVersion: \"${APP_VERSION}\"/g" ~/git/helm_deploy/sentence-planning/Chart.yaml
-            aws secretsmanager --region ${AWS_DEFAULT_REGION} get-secret-value --secret-id ${KUBE_ENV_NAMESPACE} | jq -r .SecretString | \
```
Comment (tomsmyers) [inline .circleci/config.yml:43]: i noticed that in the orb's `deploy_env` the var that is `KUBE_ENV_NAMESPACE` here is replaced with `AWS_SECRET_NAME`. do we need to make changes elesewhere to ensure this works as it used to?

File: .circleci/config.yml [hunk 2]
```diff
@@ -97,15 +45,10 @@ jobs:
       - run:
           command: |
             npm run build
-            DATE=$(date '+%Y-%m-%d')
-            export BUILD_NUMBER=${DATE}.${CIRCLE_BUILD_NUM}
-            export GIT_REF="$CIRCLE_SHA1"
```
Comment (tomsmyers) [inline .circleci/config.yml:102]: the format of the version number is slightly different here. the new value computed in the dps orb uses just the first 7 digits of the commit hash. does this matter @jonhindle ?

File: .circleci/config.yml (hunk 1 above)
Comment (mattops) [reply .circleci/config.yml:43]: We have some terraform that takes care of adding that env var to the circleci project. AWS_SECRET_NAME is there ready to use.

File: .circleci/config.yml [hunk 3]
```diff
@@ -251,35 +148,45 @@ workflows:
...
             - request-preprod-approval
       - request-prod-approval:
           type: approval
           requires:
             - deploy_preprod
-      - deploy_prod:
+      - dps/deploy_env:
+          name: deploy_prod
+          env: "prod"
+          slack_notification: true
```
Comment (tomsmyers) [inline .circleci/config.yml:188]: how do we configure this?

**Comment by tomsmyers** [comment]
waiting on merge until i've just double checked with jon about the version string.

File: .circleci/config.yml (hunk 2 above)
Comment (jonhindle) [reply .circleci/config.yml:102]: 👍 😬

**Pull request merged**